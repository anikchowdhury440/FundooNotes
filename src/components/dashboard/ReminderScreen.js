import React, {Component} from 'react'
import {View, Text} from 'react-native'

export default class ReminderScreen extends Component {
    render() {
        return (
            <View>
                <Text>Hello World</Text>
            </View>
        )
    }
}

// When we use View in child component
{/* <View>
    <View>
        <Text>Hello World</Text>
        <Text>Hello Anik</Text>
    </View>
    <View>
        <Text>Hello World</Text>
        <Text>Hello Anik</Text>
    </View>
</View> */}

//When we use Fragment in Child component
{/* <View>
        <Text>Hello World</Text>
        <Text>Hello Anik</Text>
        <Text>Hello World</Text>
        <Text>Hello Anik</Text>
</View> */}
