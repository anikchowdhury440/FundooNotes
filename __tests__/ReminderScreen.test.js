import React from 'react';
import {shallow} from 'enzyme';
import ReminderScreen from '../src/components/dashboard/ReminderScreen';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk'

const middlewares = [thunk]
const mockStore = configureStore(middlewares);
const initialState = {
    userId : '',
    userLabel : [],
    screenName : '',
    labelKey : ''
}
const store = mockStore(initialState)

describe('test ReminderScreen', () => {
    it('should match to snapshot', () => {
        const component = shallow(<ReminderScreen store = {store}/>)
        expect(component).toMatchSnapshot();
    })
})