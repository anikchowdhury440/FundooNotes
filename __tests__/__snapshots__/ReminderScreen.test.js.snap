// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ReminderScreen should match to snapshot 1`] = `ShallowWrapper {}`;
